test/index.js

```diff
@@ -0,0 +1,57 @@
+const proxyquire = require('proxyquire').noCallThru()
+const assert = require('assert')
+
+
+describe('main', () => {
+  describe('getStreams returns data from AWS', () => {
+    let main
+
+    before(() => {
```

Comment:
Thanks for looking at this for me.

I did manage to get an `aws-sdk-mock` test running (I don't know what I did different and I don't want to `git reflog` to find my original)

After seeing https://github.com/dwyl/aws-sdk-mock/issues/47 and some other problems, I guess I'll stick with what I pushed up here so far which works.  I hope to revisit `aws-sdk-mock` after it matures. This was a pretty interesting alternative approach https://github.com/dwyl/aws-sdk-mock/issues/51